pkg/provider/data_source_analytics_cluster.go

```diff
@@ -193,6 +193,23 @@ func (r *analyticsClusterDataSource) Schema(ctx context.Context, req datasource.
 					"Pausing a high availability cluster shuts down all cluster nodes",
 				Optional: true,
 			},
+			"tags": schema.SetNestedAttribute{
+				Description: "Assign existing tags or create tags to assign to this resource",
```

Comment:
as this is a data_source description, we should not mention assign or create, instead it should mention something like "show tags associated with this resource"